observability.go

```diff
@@ -39,42 +39,52 @@ var (
 
 // Opencensus stats
 var (
-	MGets            = stats.Int64("gets", "The number of Get requests", unitDimensionless)
-	MCacheHits       = stats.Int64("cache_hits", "The number of times that either cache was good", unitDimensionless)
-	MHotcacheHits    = stats.Int64("hotcache_hits", "The number of times that the hotcache cache was good", unitDimensionless)
-	MCacheMisses     = stats.Int64("cache_misses", "The number of times that either cache was not good", unitDimensionless)
-	MPeerLoads       = stats.Int64("peer_loads", "The number of remote loads or remote cache hits", unitDimensionless)
-	MPeerErrors      = stats.Int64("peer_errors", "The number of remote errors", unitDimensionless)
-	MLoads           = stats.Int64("loads", "The number of gets/cacheHits", unitDimensionless)
-	MLoadErrors      = stats.Int64("loads_errors", "The number of errors encountered during Get", unitDimensionless)
-	MLoadsDeduped    = stats.Int64("loads_deduped", "The number of loads after singleflight", unitDimensionless)
-	MLocalLoads      = stats.Int64("local_loads", "The number of good local loads", unitDimensionless)
-	MLocalLoadErrors = stats.Int64("local_load_errors", "The number of bad local loads", unitDimensionless)
-	MServerRequests  = stats.Int64("server_requests", "The number of Gets that came over the network from peers", unitDimensionless)
-	MKeyLength       = stats.Int64("key_length", "The length of keys", unitBytes)
-	MValueLength     = stats.Int64("value_length", "The length of values", unitBytes)
+	MGets              = stats.Int64("gets", "The number of Get requests", unitDimensionless)
+	MCacheHits         = stats.Int64("cache_hits", "The number of times that the cache was hit", unitDimensionless)
+	MPeerLoads         = stats.Int64("peer_loads", "The number of remote loads or remote cache hits", unitDimensionless)
+	MPeerLoadErrors    = stats.Int64("peer_errors", "The number of remote errors", unitDimensionless)
+	MLoads             = stats.Int64("loads", "The number of gets/cacheHits", unitDimensionless)
+	MLoadErrors        = stats.Int64("loads_errors", "The number of errors encountered during Get", unitDimensionless)
+	MCoalescedLoads    = stats.Int64("loads_deduped", "The number of loads coalesced by singleflight", unitDimensionless)
```

Comment:
Let's rename this metric inside the definition as well.
```suggestion
	MCoalescedLoads    = stats.Int64("loads_coalesced", "The number of loads coalesced by singleflight", unitDimensionless)
```